server/game/cards/04.2-NTB/OlFashionedHangin.js

```diff
@@ -0,0 +1,29 @@
+const ActionCard = require('../../actioncard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class OlFashionedHangin extends ActionCard {
+    setupCardAbilities() {
+        this.job({
+            title: 'Ol\' Fashioned Hangin\'',
+            playType: 'noon',
+            target: {
+                activePromptTitle: 'Choose a wanted dude to hang',
+                cardCondition: { location: 'play area', wanted: true, condition: card => card.bounty >= 2 },
+                cardType: ['dude']
+            },
+            message: context => 
+                this.game.addMessage('{0} uses {1} to lead a job to hang {2}', context.player, this, context.target),
+            onSuccess: (job, context) => {
+                this.game.resolveGameAction(GameActions.aceCard({ card: job.mark }), context);
```

Comment:
We also should check if the mark is in play before we ace it.
From the rulebook (Step 5. Is the Job Done?):
> Now resolve the effects of a successful job as given on the card, assuming the mark is still in play. If not, the job succeeds anyway but the mark is unaffected.